packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.spec.ts

```diff
@@ -73,7 +73,7 @@ describe("HorizontalScroll", () => {
         expect(element.shadowRoot?.querySelector(".scroll-next")?.classList.contains("disabled")).to.equal(false);
         
         element.style.width = "1000px";
-        await timeout(element['frameTime'] + 20);
+        await DOM.nextUpdate();
```

Comment:
I don't think this is safe. HorizontalOverflow uses setTimeout internally with element.frameTime as the timeout.

Reply:
That makes sense. I was probably just getting lucky that the resize triggered a DOM update. I've reverted it back to timeout and updated another resize to use frameTime timeout.

Reply:
@nicholasrice I updated it to use frameTime and found another case where it might be an issue and updated that too.